### PR TITLE
Adding resources for typescript curated search page

### PR DESCRIPTION
**Pull request by VontaJamal**
Made some judgement calls to put typescript lessons into buckets to start building the page.

If you have time please review @laurosilvacom

### CODE REVIEW

**Comment by laurosilvacom** [comment]
Please make PRs with appropriate resources: 
- [ ] Link to typescript page 
- [ ] Add screenshot 
- [ ] Add fun gifs

**Comment by laurosilvacom** [comment]
Good paring session @VontaJamal. 🙌  Like we mentioned during the call, the next step is to continue adding curated content.

File: src/components/search/curated/index.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import SearchJavaScript from './javascript'
 import SearchReact from './react'
 import SearchGraphql from './graphql'
+import SearchTypeScript from './typescript'
```
Comment (joelhooks) [inline src/components/search/curated/index.tsx:4]: @VontaJamal we changed how we added curated pages so I updated it here

**Review by joelhooks — APPROVED** [review]

File: src/components/search/curated/index.tsx (hunk 1 above)
Comment (VontaJamal) [reply src/components/search/curated/index.tsx:4]: understood

**Comment by VontaJamal** [comment]
@joelhooks After pairing with @laurosilvacom earlier he said to finish the curated page with this pr. You want me to continue with that plan or open a new one since you've approved this one?

**Comment by joelhooks** [comment]
you can make a new PR, just looked like it was at a good stopping point and I like to merge

--- original message ---
On February 1, 2021, 6:26 PM PST notifications@github.com wrote:



@joelhooks After pairing with @laurosilvacom earlier he said to finish the curated page with this pr. You want me to continue with that plan or open a new one since you've approved this one?

—

You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---

**Pull request merged**